### PR TITLE
Icon fixes

### PR DESCRIPTION
**Pull request by jacob314**
@terrylucas if you have higher resolution versions of these icons let me know. Checking in copies with the @2x in the same so they render on higher dpi displays.

Clearly you need a newer laptop!

### CODE REVIEW

**Comment by terrylucas** [comment]
Looks good to me sent you the 2x icons (via email attachments)

**Comment by jacob314** [comment]
Renamed the icons to match naming schemes used elsewhere (removed _1x from the name)
Fixed another unrelated icon color issue
fixed the index.html to reduce the flash of content on page load.

File: packages/devtools/web/index.html [hunk 1]
```diff
@@ -128,8 +128,8 @@
         <!-- add the known pages -->
         <a disabled><span class="octicon octicon-device-mobile"></span><span> Flutter Inspector</span></a>
         <a disabled><span class="octicon octicon-pulse"></span><span> Timeline</span></a>
+        <a disabled><span class="octicon octicon-package"></span><span> Memory</span></a>
```
Comment (terrylucas) [inline packages/devtools/web/index.html]: Cool I was wondering where this was.

File: packages/devtools/web/styles.css [hunk 2]
```diff
@@ -768,6 +768,10 @@ html [full] {
     vertical-align: middle;
 }
 
+.btn[disabled] .flutter-icon {
+    opacity: 0.5;
+}
+
```
Comment (terrylucas) [inline packages/devtools/web/styles.css]: Nice.

**Pull request merged**